src/pages/spawn/spawnVolume/SpawnVolumeTable.tsx

```diff
@@ -24,42 +25,54 @@ export const SpawnVolumeTable: React.VFC<SpawnVolumeTableProps> = ({
 }) => {
   const { search } = useLocation();
   const volume = parseQueryString(search)?.volume;
+  const dataSource: TableVolume[] = useMemo(() => {
+    const volumesCopy = [...volumes];
+    volumesCopy.sort(sortByHost);
+    const firstMigrateableVolumeId =
+      volumesCopy.find((v) => v.homeVolume)?.id ?? "";
+    return volumes.map((v) => ({
+      ...v,
+      showMigrateBtnCue:
+        v.id === firstMigrateableVolumeId &&
+        Cookies.get(SEEN_MIGRATE_GUIDE_CUE) !== "true",
+    }));
```

Comment:
(nothing to change, just a random comment) I feel like we could skip all of this work if `SEEN_MIGRATE_GUIDE_CUE` is true, but I also don't think it's a big deal since people don't usually have that many volumes